src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -630,3 +633,12 @@ export enum CypressExerciseType {
     TEXT,
     QUIZ,
 }
+
+export function parseCourseAfterMultiPart(response: Cypress.Response<Course>): Course {
```

Comment:
We often have a method convert...FromServer in the client. Maybe you could follow this naming scheme here?